backend/lp_generator.py

```
@@ -17,8 +17,9 @@
 ######################################
 
 ## geminiを使う場合
-from google import genai as genai_img
-from google.genai import types
+import google.generativeai as genai
```

Ruff (0.12.2)

Redefinition of unused `genai` from line 2

Remove definition: `genai`

(F811)